src/styles/Blocks/stars.scss

```
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  height: 16px;
+  // gap: 4px;
+  width: 96px;
+    
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    &:not(:last-child) {

```

add an empty line between selectors. Check all over the document
```suggestion
    background-position: center;
    
    &:not(:last-child) {

```